lib/orchestrator/configuration.rb

```diff
@@ -0,0 +1,35 @@
+module Orchestrator
+  class Configuration
+    include Singleton
+
+    def dynamodb_endpoint
+      case env
+      when :production
+        Exercism.config.dynamodb_endpoint
+      else
+        "http://localhost:3039"
```

Comment:
To have this work with docker-compose, we shouldmake this configurable? Something like:

```suggestion
dynamodb_host = ENV.fetch("DYNAMODB_HOST", "127.0.0.1")
"http://#{dynamodb_host}:3039"
```
"http://localhost:3039"

See https://github.com/exercism/tooling-invoker/pull/2/files#diff-6aae0d6010a1060371916ffa8eabbf14R39